Assets/Scripts/UI/CopilotUI.cs

```diff
@@ -0,0 +1,203 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+using UnityEngine.SceneManagement;
+public class CopilotUI : MonoBehaviour
+{
+
+    public static CopilotUI singleton;
+
+    private CopilotInfo selected;
+    public string sceneToLoad;
+
+    [Header("Active")]
+    #region Active
+    public Text activeDescription;
+    public Text activeName;
+    public Button activeButton;
+    public Text active;
+    public Image activeIcon;
+    public bool activeSelected;
+    #endregion
+
+    [Header("Passive")]
+    #region Passive
+    public Text passiveDescription;
+    public Text passiveName;
+    public Button passiveButton;
+    public Text passive;
+    public Image passiveIcon;
+    public bool passiveSelected;
+    #endregion
+
+    [Header("Layout")]
+    #region Layout
+    public Button start;
+    public Text selectionName;
```

Comment:
I noticed in the scene view you can't see this text because the font size is too big. Might want to adjust that